src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -172,6 +169,22 @@ public static GraphBuilder create(
         StreetLinkerModule streetLinkerModule = new StreetLinkerModule();
         streetLinkerModule.setAddExtraEdgesToAreas(config.areaVisibility);
         graphBuilder.addModule(streetLinkerModule);
+
+        if ( hasOsm ) {
+            if (OTPFeature.PruneIslands.isOn()) { // original pruning
+                PruneFloatingIslands pruneFloatingIslands = new PruneFloatingIslands();
+                pruneFloatingIslands.setPruningThresholdIslandWithoutStops(config.pruningThresholdIslandWithoutStops);
+                pruneFloatingIslands.setPruningThresholdIslandWithStops(config.pruningThresholdIslandWithStops);
```

Comment:
Could you add a comment here, I expect this to have a dependency to some of the above operations, since it is moved down after the gtfs/netx/osm section. What do this depend on?